libwallet/assets/btc/dex-wallet.go

```diff
@@ -7,166 +7,353 @@ package btc
 // https://github.com/decred/dcrdex/blob/master/client/asset/btc/spv.go
 
 import (
+	"bytes"
 	"context"
+	"encoding/hex"
+	"encoding/json"
 	"errors"
 	"fmt"
+	"math"
+	"sync"
 	"time"
 
 	"decred.org/dcrdex/client/asset"
 	dexbtc "decred.org/dcrdex/client/asset/btc"
+	"decred.org/dcrdex/dex"
+	dexbtchelper "decred.org/dcrdex/dex/networks/btc"
 	"github.com/btcsuite/btcd/btcec/v2"
-	"github.com/btcsuite/btcd/btcjson"
 	"github.com/btcsuite/btcd/btcutil"
 	"github.com/btcsuite/btcd/btcutil/gcs"
-	"github.com/btcsuite/btcd/btcutil/psbt"
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btclog"
 	"github.com/btcsuite/btcwallet/chain"
 	"github.com/btcsuite/btcwallet/waddrmgr"
 	"github.com/btcsuite/btcwallet/wallet"
 	"github.com/btcsuite/btcwallet/wallet/txauthor"
 	"github.com/btcsuite/btcwallet/wtxmgr"
+	"github.com/decred/slog"
 	"github.com/lightninglabs/neutrino"
-	"github.com/lightninglabs/neutrino/headerfs"
 )
 
-// DEXWallet wraps *wallet.Wallet and implements dexbtc.BTCWallet.
+// DEXWallet wraps *wallet.Wallet and implements dexbtc.Wallet.
 type DEXWallet struct {
-	w          *wallet.Wallet
-	acctNum    int32
-	spvService *btcChainService
+	w         *wallet.Wallet
+	acctNum   int32
```

Comment:
This is only used in `AccountInfo()` method. Do we still need that method?

Reply:
`AccountInfo()` was refactored to `accountName`